api/v1/helpers/verbs/doFind.js

```diff
@@ -68,13 +68,23 @@ function doFindAndCountAll(reqResNext, props, opts) {
  */
 function doFindAll(reqResNext, props, opts) {
```

Comment:
doFindAndCountAll (above) doesn't apply the advanced tag logic, such as filtering for multiple tags, so there are no changes required there. Is there a reason for this?